src/XrdCl/XrdClZipArchiveReader.cc

```diff
@@ -243,7 +243,7 @@ class ZipArchiveReaderImpl
     {
       delete pBuffer;
       ClearRecords();
-      pArchive.Close();
+      XRootDStatus st = pArchive.Close();
```

Comment:
Looks like to void this official way would be to add a cast to void, i.e., ` (void) pArchive.Close();` but GCC currently doesn't detect that (bug #66425), Clang does. According to  C++17 standard we have `nodiscard` which is unused-result and it says that compile should only warn/error if cast to void is missing.